### PR TITLE
Eliask.configuring db connections

### PR DESCRIPTION
**Pull request by ekraihan**
Here is my PR for removing the connection string from the source code. I put the connection string in azure instead

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -240,3 +240,5 @@ ModelManifest.xml
 
 # FAKE - F# Make
 .fake/
+
+retina-api/retina-api/Secrets.config
```
Comment (ekraihan) [inline .gitignore]: Secrets.config is for local testing so i put in in the .gitignore

File: retina-api/retina-api/Models/DBConnector.cs [hunk 2]
```diff
@@ -13,12 +14,8 @@ public class DBConnector
 
         public DBConnector()
         {
-            newConnection = new SqlConnection(
-                            "user id = renascentinc;" +
-                            "password = Skylark777!;" +
-                            "server=retinadbserver.database.windows.net;" +
-                            "database = retina_develop_db;" +
-                            "connection timeout = 30");
+            ConnectionStringSettings retinaConnectionString = ConfigurationManager.ConnectionStrings["RetinaDBConnection"];
```
Comment (ekraihan) [inline retina-api/retina-api/Models/DBConnector.cs]: This line grabs the connection string from azure

File: retina-api/retina-api/Web.config [hunk 3]
```diff
@@ -48,4 +51,9 @@
     </handlers>
   
   </system.webServer>
+  <!--
```
Comment (ekraihan) [inline retina-api/retina-api/Web.config]: I can uncomment this configuration to test locally

Comment (Jbcampbe) [inline retina-api/retina-api/Web.config]: awesome. so you can share this config file to anyone looking to test locally?

Comment (ekraihan) [inline retina-api/retina-api/Web.config]: Yes josiah

**Pull request merged**